astropy/units/quantity.py

```diff
@@ -222,8 +222,9 @@ def __new__(cls, value, unit=None, dtype=None, copy=True, order=None,
                             subok=True, ndmin=ndmin)
 
         # Maybe list/tuple of Quantity? short-circuit array for speed
-        if(not isinstance(value, np.ndarray) and isiterable(value) and
-           all(isinstance(v, Quantity) for v in value)):
+        if (not isinstance(value, np.ndarray) and isiterable(value) and
+            all(isinstance(v, Quantity) for v in value)):
```

Comment:
In cases like this I just always add another level of indentation to the continuation line:

```python
if (not isinstance(value, np.ndarray) and isiterable(value) and
        all(isinstance(v, Quantity) for v in value)):
```
Lots of Python editors do this automatically.  You'll see it everywhere if you look :)